src/test/kotlin/no/nav/klage/oppgave/service/EktefelleElasticsearchServiceTest.kt

```diff
@@ -0,0 +1,247 @@
+                versjon = 1L,
+                tildeltEnhet = "4219",
+                tema = Tema.SYK.id,
+                type = Type.KLAGE.id,
+                tildeltSaksbehandlerident = null,
+                innsendt = LocalDate.of(2018, 10, 1),
+                mottattFoersteinstans = LocalDate.of(2018, 11, 1),
+                mottattKlageinstans = LocalDateTime.of(2018, 12, 1, 0, 0),
+                frist = LocalDate.of(2019, 12, 1),
+                hjemler = listOf(),
+                created = LocalDateTime.now(),
+                modified = LocalDateTime.now(),
+                kilde = "K9",
+                temaNavn = Tema.SYK.name,
+                typeNavn = Type.KLAGE.name,
+                status = IKKE_TILDELT,
+                sakenGjelderFnr = "456"
+            )
+        val klagebehandling3 = EsKlagebehandling(
+            id = "1003L",
+            versjon = 1L,
+            tildeltEnhet = "4219",
+            tema = Tema.SYK.id,
+            type = Type.KLAGE.id,
+            tildeltSaksbehandlerident = null,
+            innsendt = LocalDate.of(2019, 10, 1),
+            mottattFoersteinstans = LocalDate.of(2019, 11, 1),
+            mottattKlageinstans = LocalDateTime.of(2019, 12, 1, 0, 0),
+            frist = LocalDate.of(2020, 12, 1),
+            hjemler = listOf(),
+            created = LocalDateTime.now(),
+            modified = LocalDateTime.now(),
+            kilde = "K9",
+            temaNavn = Tema.SYK.name,
+            typeNavn = Type.KLAGE.name,
+            status = IKKE_TILDELT,
+            sakenGjelderFnr = "123"
+        )
+        val klagebehandling4 =
+            EsKlagebehandling(
+                id = "1004L",
+                versjon = 1L,
+                tildeltEnhet = "4219",
+                tema = Tema.OMS.id,
+                type = Type.KLAGE.id,
+                tildeltSaksbehandlerident = null,
+                innsendt = LocalDate.of(2018, 10, 1),
+                mottattFoersteinstans = LocalDate.of(2018, 11, 1),
+                mottattKlageinstans = LocalDateTime.of(2018, 12, 1, 0, 0),
+                frist = LocalDate.of(2019, 12, 1),
+                hjemler = listOf(),
+                created = LocalDateTime.now(),
+                modified = LocalDateTime.now(),
+                kilde = "K9",
+                temaNavn = Tema.OMS.name,
+                typeNavn = Type.KLAGE.name,
+                status = IKKE_TILDELT,
+                sakenGjelderFnr = "456"
+            )
+        repo.save(klagebehandling1)
+        repo.save(klagebehandling2)
+        repo.save(klagebehandling3)
+        repo.save(klagebehandling4)
+
+        val query: Query = NativeSearchQueryBuilder()
+            .withQuery(QueryBuilders.matchAllQuery())
+            .build()
+        val searchHits: SearchHits<EsKlagebehandling> = esTemplate.search(query, EsKlagebehandling::class.java)
+        assertThat(searchHits.totalHits).isEqualTo(4L)
+    }
+
+    @Test
+    @Order(4)
```

Comment:
Ser at det er samme order på flere tester. Kunne man kan droppet order totalt for disse?

Reply:
Ja, jeg prøver å skrive om til å bruke @Before istedet

Reply:
Mmfff.. Jeg fikk det ikke til. Jeg endrer Order nå, og så får vi heller skrive det om til en bedre måte senere.